DEVELOPING.md

```diff
@@ -62,7 +60,9 @@ Alternatively, a report can be generated as HTML with
 pipenv run coverage html
 ```
 
-The HTML will output to `./htmlcov/` by default
+The HTML will output to `./htmlcov/` by default.
+
+The report cannot be generated on Windows directly due to the difference in file separators in the paths. Instead, launch a shell in the container (see below) and use `coverage report` or `coverage html` (do not invoke through Pipenv).
```

Comment:
Could we include the issue on the coverage docs here?

Reply:
I didn't want to because this problem is not quite the same as the one reported in the issue. I also don't foresee any useful information coming from that issue to warrant mentioning it.

Reply:
Gotcha.